dist/browser/bootlint.js

```diff
@@ -11095,6 +11104,12 @@ var LocationIndex = _location.LocationIndex;
             reporter('Using `.pull-left` or `.pull-right` as part of the media object component is deprecated as of Bootstrap v3.3.0. Use `.media-left` or `.media-right` instead.', mediaPulls);
         }
     });
+    addLinter("W011", function lintMediaPulls($, reporter) {
+        var mediaPulls = $('.navbar .pull-left, .navbar .pull-right');
```

Comment:
This doesn't seem specific enough. It's permissible to use the `.pull-*` classes within components that are within a navbar, but not on parts of the navbar itself.